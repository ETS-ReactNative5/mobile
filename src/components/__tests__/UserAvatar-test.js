import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import UserAvatar from '../UserAvatar'

it('renders correctly', () => {
  const tree = renderer.create(
    <UserAvatar avatar={'avatar'}/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
