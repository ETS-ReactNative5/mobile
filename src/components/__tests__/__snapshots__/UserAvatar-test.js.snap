exports[`test renders correctly 1`] = `
<Image
  defaultSource={1}
  source={
    Object {
      "uri": "avatar",
    }
  }
  style={undefined} />
`;
